src/hoc/Layout/Layout.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import Aux from '../Aux/Aux';
+import classes from './Layout.css';
+import Toolbar from '../../components/Navigation/Toolbar/Toolbar';
+
+class Layout extends Component {
+    render () {
+        return (
+            <Aux>
```

Comment:
Remove `<Aux>` throughout the assignment

Reply:
Need to fix this